pkg/apis/vpc/v1alpha1/subnetset_types.go

```diff
@@ -35,6 +35,9 @@ type SubnetSetSpec struct {
 
 	// Subnet DHCP configuration.
 	SubnetDHCPConfig SubnetDHCPConfig `json:"subnetDHCPConfig,omitempty"`
+	// Pre-created Subnets names owned by this SubnetSet. It's exclusive with other fields like IPv4SubnetSize, AccessMode, and SubnetDHCPConfig.
```

Comment:
For SubnetSet CR, it is extended to support user specified Subnet CR names,  those Subnet CR names can be pre-created or auto-created.

Reply:
Done